.github/workflows/docker-develop.yml

```diff
@@ -6,6 +6,7 @@ on:
       - develop
     paths-ignore:
       - 'docs/**'
+  pull_request:
```

Comment:
Why does this need to run at pull requests? We create a lot of PR with this and that wouldn't this cause issues?

Reply:
It's just for debuggin, to run on this PR. I will remove it before merging